src/lib/helpers/ripple.ts

```diff
@@ -2,7 +2,7 @@ interface RippleOptions {
   center?: boolean;
   color?: string; // CSS color
   duration?: number; // In ms
-  disable?: boolean; // Should the ripple be disabled
```

Comment:
Why are we using `disabled` instead of `disable` now?

Reply:
`disabled`

Reply:
Because is standard HTML API and follows MD3 guidelines